src/test/java/caclulator/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,48 @@
+package caclulator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringAddCalculatorTest {
+
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(result).isZero();
```

0을 비교할 땐 isZero를 활용하면 조금 더 보기 좋다고 생각해요.

Reply:
isZero 함수가 있었네요 ㅎㅎ 감사합니다